app/assets/javascripts/discourse/app/helpers/raw.js

```diff
@@ -5,16 +5,21 @@ import { RUNTIME_OPTIONS } from "discourse-common/lib/raw-handlebars-helpers";
 import { buildResolver } from "discourse-common/resolver";
 
 let resolver;
+let viewsByTemplateName = [];
```

Comment:
Wasn't this supposed to be `{}`? 😃 In any case, it would be better to use `Map` and its methods (`has`, `set`, `get`)

Reply:
Refactored to use `Map`